app/javascript/data/advancedFilterOptions.js

```diff
@@ -343,10 +343,12 @@ export const advancedFilterOptions = [
         title: 'product name',
         type: 'select',
         options: [
-          'Moderna COVID-19 Vaccine',
-          'Pfizer-BioNTech COVID-19 Vaccine',
+          'Moderna COVID-19 Vaccine (Non-US tradename: Spikevax)',
```

Comment:
if we're officially changing the name of these vaccines, we're going to have to do a migration to update the product names in the vaccines table and also the available options and selected value in the saved filters for both advanced filter and export presets

Reply:
For exports would it be valid to check first for `['data']['vaccines']['checked'].include?('product_name')` then only if that is true, edit the saved query for the product names?

Reply:
Actually, how would you query for the specific vaccine string in an export preset without using an advanced filter? Search terms are only for various IDs, right?

Reply:
you are right about it having to be queried through an advanced filter for the export preset. it pretty much works the same way for saved advanced filters and saved export presets, we just have to do updates for both (which we've done several times before in the past)

Reply:
take a look at https://github.com/SaraAlert/SaraAlert/pull/1048/commits/39ca56594662d4e90102b2f5ecf07630b2d054d8

Reply:
this seems to work, but can you write a separate `up` and `down` method so that the migration can work both ways?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1048/commits/055f4a1c7b1dcea260320bda6d6591bf644e1347